tactics/hints.ml

```diff
@@ -657,8 +657,7 @@ let searchtable_add (name,db) =
 let current_db_names () = Hintdbmap.domain !searchtable
 let current_db () = Hintdbmap.bindings !searchtable
 
-let current_pure_db () =
-  List.map snd (Hintdbmap.bindings (Hintdbmap.remove "v62" !searchtable))
+let current_pure_db () = List.map snd (current_db ())
```

Comment:
I don't know the origin of the name but it seems that it's just the hintdb you want. I am not sure what is stored with it. There's a bit more refactoring to do around this `hints.ml` file which I intend to do shortly (I was just expecting to see how this PR would get merged first). I am planning in particular to remove `current_db_names` from `hints.mli`.